cli/js/io.ts

```diff
@@ -49,13 +49,95 @@ export interface SeekerSync {
   seekSync(offset: number, whence: SeekMode): number;
 }
 
+class ReadableStreamIOReader implements Reader {
+  #reader: ReadableStreamDefaultReader;
+  #buffer: Uint8Array | null;
+  #encoder: TextEncoder;
+  constructor(dst: ReadableStream) {
```

Comment:
Shouldn't all of the `ReadableStream` and `WriteableStream` be `ReadableStream<Uint8Array>` and `WriteableStream<Uint8Array>`?

Reply:
I'm not used to set types for streams and I missed it. Should be `ReadableStream<Uint8Array | string>` and `WriteableStream<Uint8Array>`.